go-controller/pkg/config/utils.go

```diff
@@ -16,6 +16,18 @@ type HostPort struct {
 	Port int32
 }
 
+// String representation of a HostPort entry
+func (hp *HostPort) String() string {
+	switch {
+	case hp.Host == nil:
+		return fmt.Sprintf(":%v", hp.Port)
```

Comment:
Curious why you did not use '%d' format specifier?

Reply:
Just using `%v` as generic placeholder, but I can use `%d` if you feel is more precise/accurate.